src/chrome/content/rules/Tradelab.xml

```diff
@@ -1,20 +1,65 @@
 <!--
-	For problematic rules, see Tradelab-problematic.xml.
+	Cert expired:
+		- bidder2.tradelab.fr
 
+	Cert mismatch:
+		- imp.tradelab.fr
+		- pop3.tradelab.fr
+		- smtp.tradelab.fr
+		- squirrel.tradelab.fr
 
-	Nonfunctional subdomains:
 
-		- insight	(expired, self-signed, CN: www.snakeoil.dom)
+	Connection refused:
+		- audio.tradelab.fr
+		- bidder1.tradelab.fr
+		- brainx1.tradelab.fr
 
+	Self-signed cert
+		- ftp.tradelab.fr
+		- graylog.tradelab.fr
+		- help.tradelab.fr
+		- pub1.tradelab.fr
+		- wordpress.tradelab.fr
 
-	Problematic subdomains:
+	Timeout:
+		- brain3.tradelab.fr
 
-		- (www.)	(expired, self-signed, CN: plesk)
+	TLS error:
+		- calendar.tradelab.fr
+		- mail.tradelab.fr
 
 -->
 <ruleset name="Tradelab (partial)">
 
+	<target host="tradelab.fr" />
+	<target host="www.tradelab.fr" />
+
+	<target host="api.tradelab.fr" />
+	<target host="api2.tradelab.fr" />
+	<target host="backoffice.tradelab.fr" />
+	<target host="borneo.tradelab.fr" />
+	<target host="brainx2.tradelab.fr" />
 	<target host="cdn.tradelab.fr" />
+	<target host="confluence.tradelab.fr" />
```

Comment:
This requires a login. I'm not confident adding it.

Reply:
Confluence is a wiki software by Atlassian , often used by companies for their internal stuff. 
It works fine with TLS. I don't see a problem with forcing this to TLS.